requirements/specifications/lifecycle/index.md

```diff
@@ -1,274 +1,365 @@
+| Active       | ✔   | ✔   | ✔   | ✔   | ✔   | ✔   | ✔   | Full access to resources and is a perceptible part of the user experience. |
+| Suspended    | ↓   | ↓   | ✔   | ✔   |     |     |     | Reduced memory footprint and no access to graphics or A/V.                 |
+| Sleeping     |     |     |     |     |     |     |     | No CPU cycles are given to the app. App may stay in RAM or be stored.      |
+
+|   | Legend |
+| - | ------ | 
+| CPU | Central Processing Unit |
+| RAM | Ramdon Access Memory |
+| F/S | File System |
+| Net | Network |
+| GFX | Graphics rendering |
+| SFC | Graphics compositor surface |
+| A/V | Audio Video Decoder |
+| ↓ | Limited |
+| ✔ | Normal access |
+| ? | Unknown | 
 
 ### 2.1. Initializing
 
-This state allows apps to do whatever initialization is required in
-order to be a Firebolt compliant app, e.g. register for event listeners,
-etc.
+This is the initial state an app exists from the moment it starts receiving CPU cycles.
 
-When an app is first loaded the platform **MUST** consider it to be in
-the initializing state. Apps **MUST** remain in this state until they
-register with the platform via Lifecycle.ready(), or
-Lifecycle.onInactive(), or are unloaded by the platform.
+When an app starts running is **MUST** initialize the Firebolt SDK as quickly as possible
+and then wait for the `Application.create()` inteface to be called before doing further setup.
 
-The platform **MUST NOT** ever put an app back into this state once it
-has left unless it has been unloaded and then loaded again.
+Apps in this state **MUST NOT** have access to the graphics compositor.
 
-There is no notification for this state, it just is.
+Apps in this state **MUST NOT** be visible.
 
-### 2.2. Inactive
+Apps in this state **MUST NOT** receive RCU key presses.
 
-This state allows an app to be running, but not in one of the two active
-states, i.e. foreground and background. Inactive apps can execute code,
-but are not visible, do not receive any input from RCUs, and cannot use
-the video decoder. Apps should free up memory and other resources in
-this state.
+Apps in this state **MUST NOT** have access to the audio-video decoder. 
 
-When an app transitions to this state, the platform **MUST** dispatch
-the Lifecycle.onInactive notification with the current state and
-previous states.
+Apps in this state **MUST NOT** use the media pipeline.
 
-Apps in this state **MUST** not use the media pipeline and the platform
-**MUST** tear down any Media Pipeline sessions associated with this app.
+### 2.2. Running
+
+This state allows an app to be running and ready to go, but not actively part
+of the user-perceptible experience.
+
+Running apps can execute code, but are not [Presented](./presentation.md) to the user, do not receive
+any input from RCUs, and cannot use the video decoder.
+
+Apps in this state **MUST** have access to the graphics compositor.
```

Comment:
Apps in this state **MAY or SHOULD** have created, allocated a Graphics Surface but it will not be made visible by platform compositor.